aws_lambda_powertools/utilities/feature_flags/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""Advanced feature toggles utility
+"""
+from .appconfig import AppConfigStore
+from .base import StoreProvider
+from .exceptions import ConfigurationError
+from .feature_flags import FeatureFlags
+from .schema import ACTION, SchemaValidator
+
+__all__ = [
+    "ConfigurationError",
+    "FeatureFlags",
+    "ACTION",
```

Comment:
@heitorlessa - Is `ACTION` a bit to generic when imported into your own code? Or is this fine.

Reply:
yep, I've renamed to RuleAction so it also follows `EnumName` "rule"